### PR TITLE
Authenticate 

### PR DESCRIPTION
**Pull request by saragadamraviteja**
completed task1 ,2  and 3 of authentication and importing books.csv file into database as table.

### CODE REVIEW

File: project1/application.py [hunk 1]
```diff
@@ -18,24 +21,71 @@
...
+        print(password) 
+        contact = request.form.get("ContactNumber")
+        print(contact)
+        regist = User(username=name, password=password)
+        if User.query.get(name):
+            return render_template("register.html",name1=name)
+        db.session.add(regist)
+        db.session.commit()
+        return render_template("register.html",name=name)
+    return render_template("register.html")
```
Comment (manasa133) [inline project1/application.py:67]: send a message indicating successfull or not

File: project1/application.py [hunk 2]
```diff
@@ -18,24 +21,71 @@
...
+        if (obj.username == name and obj.password == password):
+            return render_template("login.html",name=name)
 
+        if(obj.username != name or obj.password != password):
+            return render_template("register.html",message="Invalid Credentials")
+    return render_template("register.html",message="Invalid Credentials")
+
+@app.route("/logout")
+def logout():
+    Session.Clear()
```
Comment (manasa133) [inline project1/application.py]: this is not the session you have to use, please check. you should be setting a session variable while logging in

File: project1/data.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+class User(db.Model):
+    __tablename__ = "details"
+    username = db.Column(db.String, primary_key = True)
+    password = db.Column(db.String, nullable = False)
+    timestamp = db.Column(db.DateTime, nullable = False)
+
+    def __init__(self, username, password):
+        self.username = username
+        self.password = password
+        self.timestamp = datetime.now()
```
Comment (manasa133) [inline project1/data.py:15]: its correct

File: project1/application.py (hunk 2 above)
Comment (saragadamraviteja) [reply project1/application.py]: changed madam..

File: project1/application.py (hunk 1 above)
Comment (saragadamraviteja) [reply project1/application.py:67]: okay madam

**Pull request closed without merging**